edge-cacher.py

```diff
@@ -118,7 +118,8 @@ def create_service_unit(config: VfsConfig, share_name: str, enable_smb: bool):
         -v \
         --config={get_rclone_config_path(share_name)} \
         --allow-other \
-        --cache-dir {config.cache_dir} \
+        --cache-dir {config.cache_dir}/cache \
+        --cache-tmp-upload-path {config.cache_dir}/tmp-upload \
```

Comment:
This is a cool feature! Thanks for updating.
Out of curiosity - did you test that in your environment? Is a sub-dir in `config.cache_dir` somehow conflicting with the rest of the operation? 
A testing harness or anything like that is still completely missing in this project (lack of time, works for me syndrome...), sorry for that.

Reply:
I didn't realize, that I still had the pull request open. This change didn't do anything, because `--vfs-cache-mode full` makes the `--cache-tmp-upload-path` obsolete. I reverted this commit.